project-base/docker/php-fpm/Dockerfile

```diff
@@ -70,3 +70,6 @@ USER www-data
 # allow configuring the GitHub OAuth token
 ARG github_oauth_token
 RUN if [ -n "$github_oauth_token" ]; then composer config -g github-oauth.github.com $github_oauth_token; fi
+
+# hirak/prestissimo makes the install of Composer dependencies faster by parallel downloading
+RUN composer global require hirak/prestissimo
```

Comment:
Can we move this package into composer.json? :)

Reply:
But I'm thinking now, If this package will be placed into composer.json, that will be too late for faster composer install. Am I right? I need to be sure.

Reply:
It is not default composer package. it is required globally, thats why it is added during dockerfile build :)